view/src/components/produce.js

```diff
@@ -499,128 +591,40 @@ class produce extends Component {
                         </Container>
 					</Dialog>
                     <Container maxWidth="lg">
-                        <Grid container spacing={2}
-                              alignItem="center">
+                        <Grid container spacing={2} alignItem="center">
                                 <Grid item xs={12}>
-										<Autocomplete
-											id="combo-box-demo"
-											options={data.map((produce) => produce.title)}
-
-											style={{ width: 300 }}
-											
-											renderInput={(params) => 
-												<TextField {...params} 
-													label="Combo box" 
-													variant="outlined" 
-													
-												/>}
-											/>
-
 									<Autocomplete
-									
-										multiple
-										id="tags-filled"
-										options={data}
-										getOptionLabel={(option) => option.name}
-										onInputChange={this.handleSearch}
-										freeSolo={true}
-										renderTags={(value, getTagProps) =>
-										value.map((option, index) => (
-											<Chip variant="standard" label={option} {...getTagProps({ index })} />
-										))
-										}
-										renderInput={(params) => (
-										<TextField {...params} variant="standard" label="search" placeholder="Search..." />
-										)}
-									/>									
-                                    <div>
-										<input type="text" value={this.state.value} onChange={this.handleSearch} />	
-                                    </div>
-									
-									<div className={classes.search}>
-									<div className={classes.searchIcon}>
-									<SearchIcon />
-									</div>
-									<InputBase
+										id="produce-name-search"
+										options={data.map((produce) => produce.name)}
+										value={value}
+										onSelect={this.handleSearch} // receive the name from data element for value
```

Comment:
```suggestion
										onSelect={this.handleSearch}  // Receive the name from data element for value
```

Reply:
(direct commit isn't working from the prettier-run commit, resolved on my own)